tensorrt_llm/llmapi/trtllm-llmapi-launch

```diff
@@ -10,8 +10,9 @@ log_stderr "mpi_rank: $mpi_rank"
 
 pid=$(ps -o pid= -p $$ | tr -d ' ')
 
-# Tell TRTLLM to spawn a additional process for the Proxy
-export TLLM_SPAWN_PROXY_PROCESS=1
+# A file to share the IPC address between the server and client
+export TLLM_SPAWN_PROXY_PROCESS_ADDR_FILE=/tmp/llmapi-proxy-addr-${pid}
+rm -f $TLLM_SPAWN_PROXY_PROCESS_ADDR_FILE
```

Comment:
if we use a tmpfs like /dev/shm, not /tmp. Maybe you don't need to delete it? Because it won't exists?